src/i18n/Localizable.xcstrings

```diff
@@ -23333,6 +23454,7 @@
       }
     },
     "Suggest" : {
+      "comment" : "is this one used as verb or noun?",
```

Comment:
Verb

Reply:
is this an action invoked by user? for example, "suggest" to somebody. or is this a sort of suggestion given by WebSSH?  
the former would use gerund, and noun for the latter.